app/src/main/java/org/oppia/app/splash/SplashActivity.kt

```diff
@@ -0,0 +1,27 @@
+package org.oppia.app.splash
+
+
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.FragmentTransaction
+
+import android.os.Bundle
+import android.view.WindowManager
+import com.example.myapplication.SplashFragment
+import org.oppia.app.R
+
+class SplashActivity : AppCompatActivity() {
```

Comment:
Please add a Javadoc comment explaining what this activity is. Ditto for the fragment below.

Reply:
Added JavaDoc comments.